basis/common-modelmapper/src/main/java/com/alexkoveckiy/common/modelmapper/ModelMapperService.java

```diff
@@ -0,0 +1,24 @@
+package com.alexkoveckiy.common.modelmapper;
+
+import com.alexkoveckiy.common.dao.entities.ProfileStatusEntity;
+import com.alexkoveckiy.common.dto.configuration.DTO.ProfileStatusDTO;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by alex on 22.03.17.
+ */
+@Component
+public class ModelMapperService extends ModelMapper {
```

Comment:
@Alexey007gold You should not extend model mapper. You should make wrapper service.